src/crimson/os/seastore/seastore_types.h

```diff
@@ -711,6 +711,110 @@ struct rbm_alloc_delta_t {
   op_types_t op;
 };
 
+struct extent_info_t {
```

Comment:
nit, instead of "add"

> crimson/os/seastore: add record related code to a common place instead of Journal

might want to put "extract", like

> crimson/os/seastore: extract record related code into seastore_types.{h,cc}

so it's more specific.

Reply:
will do

Reply:
done:-)